### ISSUE TITLE
small error

### ISSUE THREAD

**Issue by fugazi73** [coldfrontlabs/fserver#33]
after install you get a page not found in the panels. The projects are also not linked in views. Just looking for info and further bug.

![07-06-_2014_10-41-31](https://cloud.githubusercontent.com/assets/7265472/3208183/1e07f0d2-ee20-11e3-8ad2-8ce4a74007d5.jpg)
![07-06-_2014_10-42-23](https://cloud.githubusercontent.com/assets/7265472/3208184/1e081224-ee20-11e3-8b1d-d54cb5708a05.jpg)
![07-06-_2014_10-42-32](https://cloud.githubusercontent.com/assets/7265472/3208182/1e07a2e4-ee20-11e3-8843-9f4dd3b7d4bd.jpg)
![07-06-_2014_10-42-55](https://cloud.githubusercontent.com/assets/7265472/3208185/1e086256-ee20-11e3-8f70-4a20b2abddca.jpg)
![07-06-_2014_10-43-35](https://cloud.githubusercontent.com/assets/7265472/3208186/1e087930-ee20-11e3-904c-b8dc32cdae83.jpg)
![07-06-_2014_10-43-51](https://cloud.githubusercontent.com/assets/7265472/3208187/1e089fdc-ee20-11e3-8ff4-1ee0b4533c88.jpg)

**Comment by minorOffense** [comment]
Yeah we removed ECK which probably means our Views are broken in fserver_ui. Merge requests are more than welcome.

We have our own UI built for the project we're using fserver with so we won't get back to this UI for about a month (I have a new employee starting, will probably one of his first tasks).

But if you can get the basic views up again, I'll merge them in.

**Comment by fugazi73** [comment]
Thank you for the note, I'll wait until I can use views. I am pleased that this module is very well maintained. I can not wait to use it.
Many Thanks
Best Regards

**Comment by fugazi73** [comment]
views still shows Broken/missing handler (Name) just for info
